drivers/char/tpm/Kconfig

```diff
@@ -86,6 +86,18 @@ config TCG_TIS_I2C
 	  To compile this driver as a module, choose M here;
 	  the module will be called tpm_tis_i2c.
 
+config TCG_TIS_I2C_ST33HTPM
```

Comment:
@rouming , it turned out we don't need this port because the TPM of the EPC E3720 device works with the >= 6.1.38 i2c's generic TPM driver. If you look to our kernel, this commit was reverted recently and the support for this driver removed, but now you are introducing it again. Since you are rebasing your 6.1.106 branch, you can safely remove all commits related to this driver (including the reversion)....